sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server/README.md

```diff
@@ -3,109 +3,102 @@
-
-### Register the web API
-
-1. Go to the Microsoft identity platform for developers App registrations portal.
+### Configure web api
+1. Search for and select your tenant in **Azure Active Directory**.
+2. Under Manage In the same tenant, select **App registrations** -> **New registration**.![Protal manage](docs/image-protal-manage.png "Protal manage")
+3. The registered application name is filled into `webapi`, select **Accounts in this organizational directory only**, click the **register** button.![Register a web api](docs/image-register-a-web-api.png "Register a web api")
+4. Under **webapi** application, select **Certificates & secrets** -> **new client secret**, expires select **Never**, click the **add** button.(Remember to save the secrets here and use them later.)![Creat secrets](docs/image-creat-secrets-api.png "Creat secrets")
+5. Under **webapi** application, select **Expose an API** -> **Add a scope**, Use the default Application ID URI, click **Save and continue** button.![Set application id url](docs/image-set-application-id-url.png "Set application id url")
+6. After step five,the page will refresh again. Then set the **Scope name** to `File.read`.(This scope will be used in application.yml.)![Add a app id url](docs/image-add-a-scope.png "Add a app id url")
+7. Finally, the api exposed in `webapi`.![Finally, the API exposed in webAPI](docs/image-expose-api.png "Finally, the API exposed in webAPI")
 
-2. Select New registration.
-    ![Select New registration](docs/image-select-new-registration.png "Select new registration")
+See [Expose scoped permission to web api] for more information about web api.
 
-3. When the Register an application page opens, enter your application's registration information:
-    ![Scope Config](docs/image-register-an-application.png "Register an application")
-    
-4. In the **Expose an API** section, select **Add a scope**, accept the proposed Application ID URI `(api://{clientId})` (back up the Application ID URI here, which will be used in the properties file) by selecting **Save and Continue**.
-   ![App-Id-Uri Config](docs/image-app-id-uri-config.png "App-id-uri Config")
-   Then enter the following information:
-   - For **Scope name**, enter **File.read**.
-   - For **Who can consent**, ensure that the **Admins and users** option is selected.
-   - In the **Admin consent display name** box, enter **Access File.read as a user**.
-   - In the **Admin consent description** box, enter **Accesses the File.read web API as a user**.
-   - In the **User consent display name** box, enter **Access File.read as a user**.
-   - In the **User consent description** box, enter **Accesses the File.read web API as a user**.
-   - For **State**, keep **Enabled**.
-   - Select **Add scope**.
-   ![Scope Config](docs/image-scope-configurations.png "Scope Config")
+### Configure web app
+If you don't know web app at the moment, you can look at this [azure-spring-boot-sample-active-directory-webapp].
```

Comment:
"If you don't know web app at the moment, you can look at this [azure-spring-boot-sample-active-directory-webapp]."
--->
"If you are not familiar with web app , you can refer this [azure-spring-boot-sample-active-directory-webapp]."